src/components/profile/Profile.vue

```diff
@@ -504,9 +503,17 @@ export default {
   border-radius: var(--cardRadius);
   position: relative;
 }
+
+.profile__section.first {
+  padding: 0;
+}
+
+.profile__section.first .profile__section-header {
+  margin: 0 0 1.5em 0;
+}
 @supports (--key: value) {
   .profile__section {
-    border-color: transparent;
+    border: none;
```

Comment:
This kills the blue border in edit mode, doesn't it?

Reply:
Hmm..yes, but there already wasn't last I checked. I might have messed that up with my other changes though. I see the blue border in the mockups. I'll adjust for that.